docs/content/docs/test/Runner.md

```diff
@@ -525,6 +525,26 @@ instrumentation {
 ./gradlew instrumentationLocal -PincludedAnnotation="package.AnotherAnnotationClassName" 
 ```
 
+## Find out how filters applied
+
+If build finished successfully It will produce files in which you must find filters debug information
+Files will be located at: \
+ `<Project root folder>/outputs/<subproject name>/instrumentation/<instrumentation task name>/filter`
+
+### Find out what filter config was
+
+Look at file `filter-config.json`
```

Comment:
Can we add some details about the content? Maybe a small example.

Reply:
It's too simple. If we add some detail we must support those details. I don't want to support those files behavior

Reply:
Ok